b827ebfffe9df797.json

```diff
@@ -0,0 +1,17 @@
+"gateway_conf": {
+    "gateway_ID": "b827ebfffe9df797",
```

Comment:
You need to use upper case for the ID, and also for the filename (not the `.json` extension thou)